### PR TITLE
Tutorial mobile

### PR DESCRIPTION
**Pull request by bc163836**
- Fixed Tutorial Mobile View
- Updated Visuals slightly for Login Page

### CODE REVIEW

**Review by sindhufelicia — APPROVED** [review]
Some changes on the tutorial page (HTML and CSS part) for the mobile template. Looks good to me.

**Comment by CelynChew** [comment]
wait for tests to be merged before merging

**Pull request merged**
